.conda-envs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: "pyatlan"
+  version: "{{ environ['GIT_DESCRIBE_TAG'] }}"
+
+source:
+  path: ../../
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pip
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
```

Comment:
Just one doubt here: It seems the `requirements.hosts` should match `requirements.txt` ([we should specify `lazy_loader~=0.4`](https://github.com/atlanhq/atlan-python/blob/main/requirements.txt)). 

I also noticed some extra spaces ` `, such as in `requests >=2.24` (though it still works)

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run